lib/open_project/journal/attachment_helper.rb

```diff
@@ -35,6 +35,13 @@ def attachments_changed(_obj)
           save
         end
       end
+
+      def file_links_changed(_obj)
+        unless new_record?
+          add_journal
+          save
+        end
+      end
```

Comment:
This part is probably not needed. It is never referenced anywhere AFAIK

Reply:
Yup, it was a leftover from my previous quackery.